File: py/8.py
# challenge 4-b: Write a program in Python that stores student names and their grades in a dictionary, and prints each student's name and grade.

students = {
    "Sami Achnine": 11,
    "Serena Rogers": 11,
    "Rahul Murgai": 12,
    "Gerald Robotnik": 95,
}

for student in students:
    print(student + "\ngrade: " + str(students[student]))

File: js/8.js
const students = new Map();

students.set("Sami Achnine", 11);
students.set("Serena Rogers", 11);
students.set("Rahul Murgai", 12);
students.set("Gerald Robotnik", 95);

for (const x of students.values()) {
  console.log(x);
}